packages/react-aria-components/docs/DropZone.mdx

```diff
@@ -0,0 +1,397 @@
+{/* Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:react-aria-components';
+import typesDocs from 'docs:@react-types/shared/src/events.d.ts';
+import {PropTable, HeaderInfo, TypeLink, PageDescription, StateTable} from '@react-spectrum/docs';
+import styles from '@react-spectrum/docs/src/docs.css';
+import packageData from 'react-aria-components/package.json';
+import Anatomy from '@react-aria/datepicker/docs/daterangepicker-anatomy.svg';
+import ChevronRight from '@spectrum-icons/workflow/ChevronRight';
+import {Divider} from '@react-spectrum/divider';
+import {ExampleCard} from '@react-spectrum/docs/src/ExampleCard';
+import {Keyboard} from '@react-spectrum/text';
+
+---
+category: Drag and drop
+keywords: [dropzone, drop, dnd, drag and drop, aria, accessibility]
+type: component
+---
+
+# DropZone
+
+<PageDescription>{docs.exports.DropZone.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['DropZone']} />
+
+## Example
+
+```tsx example
+import {DropZone, Text} from 'react-aria-components';
+
+function Example() {
+  let [dropped, setDropped] = React.useState(false);
+
+  return (
+    <DropZone
+      onDrop={() => {
+        setDropped(true);
+      }}>
+      {dropped ? "You dropped something" : "Drop object here"}
```

Comment:
Did you mean to wrap this in `<Text slot="label">`? I see it's imported above.

Reply:
oops yeah